### ISSUE TITLE
Blind rolls not being hidden

### ISSUE THREAD

**Issue by Avitale34** [dddice/dddice-foundry-plugin#3]
Even with the setting on blind rolls are being shown to players when they roll.

**Comment by CelesteBloodreign** [comment]
Thanks for reaching out and sorry this isn't working for you!

Would you be able to give a few more details on what you are experiencing so that I may better reproduce this on my side?

When I try something like this:
![image](https://github.com/dddice/dddice-foundry-plugin/assets/116366998/a35b92ac-d961-4613-b0c4-3cf8c0ff61f1)

The rolling player doesn't see it, unless they are also a GM.

I presume you are initiating this roll in a different way?

**Comment by Avitale34** [comment]
Yes I am having the players roll off their sheets, normally their skill rolls are sent blind. They can't see the result in chat but can see the 3d dice roll across their screen with what they rolled.

**Comment by Avitale34** [comment]
in past versions they came up as ? on the dice faces.

**Comment by CelesteBloodreign** [comment]
Which game system are you using? Is this dungeons and dragons?
